nstars.c

```diff
@@ -157,7 +157,7 @@ int main(int argc, char *argv[])
 					beep();
 				}
 				break;
-			case 10:					/* Enter */
+			case KEY_ENTER:
```

Comment:
Oh, terminals. I will add another commit, but I am thinking to use `'\n'` and `'\r'`[escapes][e] instead of the ASCII values, which probably would be moved into comments. What do you think?

[e]: https://en.wikipedia.org/wiki/Escape_sequences_in_C#Table_of_escape_sequences

 I am using Linux with st, I want to know what yours are, so if I stumble on this, I would know where to look into.